src/components/block-gallery/styles/style/_captions.scss

```diff
@@ -1,5 +1,5 @@
 .has-caption-style-light {
-	.coblocks-gallery--item .coblocks-gallery--figure figcaption {
```

Comment:
Removed class `.coblocks-gallery--figure` specification for support of Offset block. No visual regressions detected.